serverless/event_sources/knative-event-sources.adoc

```diff
@@ -0,0 +1,30 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="knative-event-sources"]
+= Getting started with event sources
+include::modules/common-attributes.adoc[]
+:context: knative-event-sources
+toc::[]
+
+An event source is a resource object that acts as a link between an event producer and a _sink_. A sink can be a Knative Service, Channel, or Broker that receives Events from an event source.
+
+Currently, {ServerlessProductName} supports the following event source types:
+
+ApiServerSource:: Connects a sink to the Kubernetes API server.
+PingSource:: Periodically sends ping Events with a constant payload. It can be used as a timer.
+
+SinkBinding is also supported, which allows you to connect core Kubernetes resources such as `Deployment`, `Job`, or `StatefulSet` with a sink.
```

Comment:
We haven't been doing these in code font. I would just leave them as Deployment, Job, and StatefulSet.

Reply:
I actually wondered about this, thanks.